test/sendEmail.test.js

```diff
@@ -0,0 +1,18 @@
+const sendTestEmail = require('../mailing/sender.js');
```

Comment:
When navigating files and directories, a single dot (`./mailing/sender.js`) means: "check the current working directory". The working directory, is where our command is running from (so in this example, since `sendEmail.test.js` is running in the `test` folder, it would look for a file in the same folder: `<backend repo root>/test/mailing/sender.js`)

Then two docs (`../mailing/sender.js`) means "check the parent of the current working directory"). So node would go one folder up from `test` which would bring it to the root of the repo folder. It would try to find: `<backend repo root>/mailing/sender.js`). In this case, we do not have a file there.

That's why we get the following error:

```
Cannot find module '../mailing/sender.js' from 'test/sendEmail.test.js'
```

So if we are starting in the `test` folder but we want to `<backend repo root>/src/mailing/sender.js`: [https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/test-emails/src/mailing/sender.js](https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/test-emails/src/mailing/sender.js)

We'd need to go up one folder, then into `src`, then into `mailing`. So up one folder == two dots; then we just add the path to the file:

```suggestion
const sendTestEmail = require('../src/mailing/sender.js');
```